deployments/warp_routes/EZETHSTAGE/arbitrum-base-blast-bsc-ethereum-fraxtal-linea-mode-optimism-sei-swell-taiko-zircuit-deploy.yaml

```diff
@@ -978,10 +966,7 @@ ethereum:
   isNft: false
   mailbox: "0xc005dc82818d67AF737725bD4bf75435d065D239"
   owner: "0xA9421c6F339eC414b7e77449986bE9C2Ae430C25"
-  proxyAdmin:
-    address: "0x2F78F22a1D7491500C9ED9352b8239fbAbcDd84E"
-    owner: "0xD1e6626310fD54Eceb5b9a51dA2eC329D6D4B68A"
-  token: "0xC8140dA31E6bCa19b287cC35531c2212763C2059"
+  token: "0x74c8290836612e6251E49e8f3198fdD80C4DbEB8"
```

Comment:
Is this expected?

Reply:
yes, i realized after deployment that we use the production lockbox which was fixed by calling setLockbox()